x-pack/plugins/security_solution/public/dashboards/components/dashboard_renderer.tsx

```diff
@@ -73,20 +103,28 @@ const DashboardRendererComponent = ({
     dashboardContainer?.updateInput({ timeRange, query, filters });
   }, [dashboardContainer, filters, query, timeRange]);
 
-  const handleDashboardLoaded = useCallback(
-    (container: DashboardAPI) => {
-      setDashboardContainer(container);
-      onDashboardContainerLoaded?.(container);
-    },
-    [onDashboardContainerLoaded]
-  );
-  return savedObjectId && canReadDashboard ? (
-    <DashboardContainerRenderer
-      ref={handleDashboardLoaded}
-      savedObjectId={savedObjectId}
-      getCreationOptions={getCreationOptions}
-    />
-  ) : null;
+  useEffect(() => {
+    if (isCreateDashboard && firstSecurityTagId)
+      dashboardContainer?.updateInput({ tags: [firstSecurityTagId] });
+  }, [dashboardContainer, firstSecurityTagId, isCreateDashboard]);
+
+  useEffect(() => {
+    if (wrapperRef) {
```

Comment:
shouldn't we check the reference value `wrapperRef.current` instead of the reference itself?